app/models/bulkrax/oai_adventist_set_entry.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module Bulkrax
+  class OaiAdventistSetEntry < OaiSetEntry
+    # override to swap out the thumbnail_url
+    def build_metadata
+      self.parsed_metadata = {}
+      # rubocop:disable Style/RedundantSelf
+      #
+      # Yes I could favor `parsed_metadata[work_identifier]` however in the above line, we use
+      # `self.` And so as to maintain the visual cues of "these are the same symbols" I'm choosing
+      # to disable the RedundantSelf cop for this method.
+      self.parsed_metadata[work_identifier] = [record.header.identifier]
+      self.raw_metadata = { record_level_xml: record._source.to_s }
+
+      record.metadata.children.each do |child|
+        child.children.each do |node|
+          add_metadata(node.name, node.content)
+        end
+      end
+
+      # see app/models/concerns/bulkrax/has_local_processing.rb
+      add_local
```

Comment:
No visibility, rights statement or adminset? See the qdc method which closely mirrors this one. I think my underlying question might be “why is this different from that one?”

Reply:
I was basing this on the Bulkrax::OaiSetEntry, which has the following build_metadata:

```ruby
    def build_metadata
      self.parsed_metadata = self.raw_metadata
      add_local
      return self.parsed_metadata
    end
```

That said, I can definitely add those in.  Digging deeper into Bulkrax, we have parallel hierarchies (XML, CSV, OAI) each with different opinions of how we parse data and what we assign.  The OAI side of bulkrax and cousin XML appear to be conceptually divergent.  This is likely an upstream problem.

However, I will correct this in Adventist.